Resources/Prototypes/Entities/Structures/base_structure.yml

```
@@ -32,7 +32,7 @@
     damagePopupType: SmallFloating
     isStructure: true
   - type: TTS
-    voice: FactCore # Sunrise-TTS
+    voice: Portal2FactCore # Sunrise-TTS
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)